Makefile

```diff
@@ -133,17 +144,16 @@ ifeq ($(SKIP_DOWNLOAD_EXECUTABLE),true)
 	@echo "Skipping download of sidecar executable since it already exists at $(EXECUTABLE_DOWNLOAD_PATH)"
 else
 	mkdir -p bin && \
-	echo "Using curl to download sidecar executable from GitHub release $(IDE_SIDECAR_VERSION)"; \
-	export EXECUTABLE_PATH=ide-sidecar-$(IDE_SIDECAR_VERSION_NO_V)-runner-$(SIDECAR_OS_ARCH) && \
```

Comment:
Moved up under the windows check so we can update `EXECUTABLE_DOWNLOAD_PATH` at the same time